PHASES.md

```diff
@@ -1,155 +1,130 @@
-# How to Cut a Release
 
-We release the components of Knative every 6 weeks. All of these components must
-be moved to the latest "release" of all shared dependencies prior to each
-release.
+# Phases of a Knative release cycle
 
-For details on how to cut a specific repository, see the [release a repository](#release-a-repository) section of this document.
+We release each repository of `knative.dev` every 6 weeks. Please check the [release schedule](README.md#release-schedule) to calculate when to start work on each phase of a release cycle.
 
-## Schedule of Tasks
+## Repos to be released
+- All repos to be released are listed in this document, broken down by phases
+- Each repo indicates its releasability and other statuses, when applicable
+- Each repo needs to be successfully updated to use the latest version of all shared dependencies **before** cutting a release branch
 
-* [First week of the rotation](#first-week-of-the-rotation)
-* [14 days prior to the release](#14-days-prior-to-the-release)
-* [7 days prior to the release](#7-days-prior-to-the-release)
-* [1 day prior to the release](#1-day-prior-to-the-release)
-* [Day of the release](#day-of-the-release)
-* [Post-Release work](#post-release-work)
+## Checklists
+- The cheklist in each phase helps to ensure that all work needed for a release is completed, and done in the proper order
+- Many checklist items require performing various checks and other documented [procedures](PROCEDURES.md)
+- For each phase, open its corresponding GitHub checklist issue at the start of the work
+- Please do not open a new checklist before being done with the one that comes before
 
-## First week of the rotation
+## Phases
+   - [T-minus 30 days](#t-minus-30-days)
+   - [T-minus 14 days](#t-minus-14-days)
+   - [T-minus 7 days](#t-minus-7-days)
+   - [T-minus 1 day](#t-minus-1-day)
+   - [T-minus zero - day of the release](#t-minus-zero---day-of-release)
+    - [T-minus zero first batch](#t-minus-zero-first-batch)
```

Comment:
I think the tradeoff here is precision vs readability and usability for humans. Given all the manual checks that need to be done, might not be a bad idea to have additional grouping (phases) that can be tackled and verified in smaller pieces, even if they could be combined into one bigger one.

Reply:
I **think** that was the spirit of the original docs. But I also see a benefit of the "rolling horizon" idea, which would give the green light for work to start on the "next thing", rather than holding up until a batch is done.

Hm... now I don't know. 🙂